src/main/java/book/store/dto/request/CreateBookRequestDto.java

```diff
@@ -0,0 +1,21 @@
+package book.store.dto.request;
+
+import jakarta.validation.constraints.NotNull;
+import java.math.BigDecimal;
+import lombok.Data;
+import org.springframework.stereotype.Component;
+
+@Data
+@Component
```

Comment:
never use annotations if you dont understand how they work
```suggestion
```

Reply:
@sokimaaa I know that this annotation mark the beans in Spring, this annotation is the "parent" for @Service, @Repository, @Controller annotations. 
This annotation used in video about DTO for DTO.
Please, explain why I must not use @Component and what annotation I have to use instead last one, I didn't find this information.